client/src/components/TripleValue.js

```diff
@@ -0,0 +1,104 @@
+import React from "react"
+import { TimeSeries, TimeRange, Index } from "pondjs";
+
+import {
+    Charts,
+    ChartContainer,
+    ChartRow,
+    YAxis,
+    LineChart,
+} from "react-timeseries-charts"
+
+class TripleValue extends React.Component {
```

Comment:
You have different components for having different # of channels (columns) displayed; perhaps you can generalize them into a single component